$OpenBSD: patch-helper-scripts_prespawn,v 1.2 2014/06/24 21:45:06 jeremy Exp $

Just because you can listen on 0.0.0.0 does not mean you can
connect to it.

--- helper-scripts/prespawn.orig	Thu May 29 05:13:25 2014
+++ helper-scripts/prespawn	Thu Jun  5 09:02:32 2014
@@ -80,7 +80,9 @@ class PrespawnLocation
 	end
 
 	def request_host
-		@uri.host
+		h = @uri.host
+		h = '127.0.0.1' if h == '0.0.0.0'
+		h
 	end
 
 	def socket
